src/store/actions/actions-model.ts

```diff
@@ -126,6 +130,18 @@ export const setRules = ({ commit }, rules: any) => {
   commit('mutateRules', rules)
```

Comment:
Use type "CreateRulesIF" for rules on line 129.

Reply:
updated to use `CreateRulesIF`